javascript/question/app.js

```diff
@@ -0,0 +1,163 @@
+//Q1 変数
+
+let nickname ='あらし';
+let age = 31;
+
+console.log('私のニックネームは' + nickname + 'です。年齢は'+ age + 'です');
+
+//Q2 配列
+
+const languages = ["JavaScript", "PHP", "Ruby", "Python", "Go"];
+console.log(`私の好きな言語は${languages[0]}です。次は${languages[3]}を勉強してみたいです。`);
+
+// Q3 オブジェクト
+
+let user = {
+    name: 'John',
+    age: 26,
+    bloodType: 'A',
+    favorite: 'card',
+  };
+  console.log(user.age);
+
+//Q4 配列✖︎オブジェクト
+let playerList = [
+    {
+        name: 'John',
+        age: 26,
+        favorites: ['Card Game', 'Basket Ball', 'Programming'],
+    },
+    {
+        name: 'Bob',
+        age: 33,
+        favorites: ['Tinder', 'The Legend of Zelda'],
+    },
+    {
+        name: 'Michael',
+        age: 22,
+        favorites: ['Football', 'Smash Bros.'],
+    },
+    ];
+
+    console.log(playerList[1].favorites[1]);
+
+//Q5 四則演算
+console.log((playerList[0].age + playerList[1].age + playerList[2].age) / 3);
```

Comment:
基礎問題Q5

このように`.length`というプロパティを使うと、配列やオブジェクトの中身の要素の個数を取得することができます。
こうしておくと、中身の数が**3**個じゃなくなっても、中身の個数の値を変更する必要がないため、メンテナンスが楽に（＝保守性が高く）なります。
```suggestion
//Q5 四則演算
console.log((playerList[0].age + playerList[1].age + playerList[2].age) / playerList.length);
```

Reply:
playerList.lengthで合計人数を自動化するように、変更しました。